.nais/obo-nais-dev.yaml

```diff
@@ -46,6 +46,9 @@ spec:
           "header": {
             "csp": {
               "connectSrc": ["'self'", "wss:", "*.nav.no", "*.adeo.no"]
+             },
+            "corp": {
+            "policy": "cross-origin"
```

Comment:
dette er bare for pirk: kanskje ha et lite innrykk her? (om ikke det bare er github sin visning som lurer meg)